test/data/ansible/check_ostree.yaml

```diff
@@ -329,8 +329,7 @@
         - assert:
             that:
               - result_dmesg_error.stdout_lines | length == 2
-              - "'pcieport 0000:00:01.6: pciehp: Failed to check link status' in result_dmesg_error.stdout"
-              - "'RAS: Correctable Errors collector initialized' in result_dmesg_error.stdout"
+              - "'pcieport 0000:00:01.6: pciehp: Failed to check link status' in result_dmesg_error.stdout or 'RAS: Correctable Errors collector initialized' in result_dmesg_error.stdout or result_dmesg_error.stdout == ''"
```

Comment:
This was raised recently on another PR, IIRC by @ondrejbudai. 

At this point do we even care to check dmesg output ? It starts to feel a bit fragile. 

need_info @henrywang, @yih-redhat

Reply:
@henrywang why make #1564 as a separate PR instead of reusing the current PR and/or have @yuxisun1217 just delete the relevant lines ?

Reply:
FTR I also don't see a public discussion about the previous question.

Reply:
I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.

Reply:
I don't follow here. Why not introduce the deletion as a separate commit in this pull request instead of opening new PR and having to wait for it before continuing work on this one ?

Reply:
> I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.

I'm sorry but I'm not sure what kind of reply is expected from me. :D

Reply:
Hmmm I think we still can see the previous change in this conversation, although it is marked as outdated. It will be a little complex to merge if I create multiple commit in one PR and many maintainers don't like that I think.

Reply:
> It will be a little complex to merge if I create multiple commit in one PR and many maintainers don't like that I think.

This is quite the opposite, we rarely have pull requests containing a single commit only. Most of them have quite a few commits. The governing principle is that those commits are logically related to achieve the functionality for which the PR was opened.

Reply:
Understand. I'll be aware of that the next time. Thanks @atodorov !

Reply:
> > I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.
> 
> I'm sorry but I'm not sure what kind of reply is expected from me. :D

Hi @ondrejbudai ,
We may need you to confirm "At this point do we even care to check dmesg output ? It starts to feel a bit fragile." (see the first comment by Alex, if this checkpoint is added by you :D )

Reply:
Ah, I can't remember, sorry. I am all for removing these checks, testing log output can be tricky.

Reply:
OK thanks @ondrejbudai ! 

Hi @atodorov , could you please help to merge this PR? Thanks!